### PR TITLE
[receiver/vcenter] add statement of support for version 8 of ESXi and vCenter

### PR DESCRIPTION
**Pull request by atoulme**
**Description:**
add statement of support for version 8 of ESXi and vCenter, following https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/23769

### CODE REVIEW

File: .chloggen/add_supported_version_8.yaml
Comment (dmitryax) [inline .chloggen/add_supported_version_8.yaml:27]: Do we really need a changelog entry for the doc update? Was the added support captured here before?

Comment (atoulme) [reply .chloggen/add_supported_version_8.yaml:27]: It wasn't captured before, I expect it to be significant enough it warrants a changelog entry but I'd defer to the codeowners.

**Review by dmitryax — APPROVED** [review]

**Review by djaglowski — APPROVED** [review]

**Pull request merged**
